evap/results/exporters.py

```diff
@@ -124,52 +138,58 @@ def filter_evaluations(semesters, evaluation_states, degrees, course_types, cont
                 continue
             if not evaluation.can_publish_rating_results and not include_not_enough_voters:
                 continue
-            results = OrderedDict()
+            results: OrderedDict[int, list[QuestionResult]] = OrderedDict()
             for contribution_result in get_results(evaluation).contribution_results:
                 for questionnaire_result in contribution_result.questionnaire_results:
                     # RatingQuestion.counts is a tuple of integers or None, if this tuple is all zero, we want to exclude it
+                    question_results: list[QuestionResult] = questionnaire_result.question_results
                     if all(
-                        not question_result.question.is_rating_question or not RatingResult.has_answers(question_result)
-                        for question_result in questionnaire_result.question_results
+                        not isinstance(question_result, RatingResult) or not RatingResult.has_answers(question_result)
```

Comment:
Shouldn't this be the same now?
```suggestion
                        not isinstance(question_result, AnsweredRatingResult)
```
If so, I think `not any(isinstance(...))` would be nicer to read now :)

Reply:
did include it

Reply:
Again, I saw that you committed the initial suggestion, but you did not respond to my suggestion of using `not any` instead. I still think "none of the them are answered" is nicer than "all of them are un-answered"

I wouldn't have wrote a comment asking you to re-check the discussions if there wouldn't be something still left in those :)